### ISSUE TITLE
Enable automatic configuration of sublayers

### ISSUE THREAD

**Issue by ChristophMaskos** [open-pioneer/trails-openlayers-base-packages#380]
### Issue description

Currently, each sublayer of a WMS-service must be configured individually. This is impractical for WMS -Layers that have more than just a few sublayers.

Here is an example of a WMS-service that has a single sublayer per year:
 [Link](https://www.wms.nrw.de/geobasis/wms_nw_hist_dop?Service=WMS&Request=GetCapabilities)

This is not only tiring, but also increases the size of the code base unnecessarily.

The default case should be that all sublayers are configured, unless you have specified an array of sublayers in the configuration. This should apply to all types of layers with sublayers. But most urgently for WMS Layer.

### Validations

- [X] Read and follow our [code of conduct](https://github.com/open-pioneer/.github/blob/main/CODE_OF_CONDUCT.md)
- [X] Read the [contributing guidelines](https://github.com/open-pioneer/trails-starter/blob/main/docs/Contributing.md)
- [X] Check that there isn't already a similar issue

**Comment by mbeckem** [comment]
Agreed, this would be welcome improvement. We're not likely to work on that right now during maintenance (focusing on chakra v3 and other technical issues). This will likely have to be an outside contribution or part of a future feature sprint.

## Notes

Example syntax:

```
new WMSLayer({
    sublayers: "*" // also the default
});
```

- Requires fetching and parsing WMS capabilities (to figure out which sublayers exist) when the layer is created
    - Note: fetching can only happen after the map has been attached to the map, so it cannot be started from the layer's constructor
    - Should support cancellation (abort triggered from `destroy()`).
- After capabilities have been parsed, the `layer.sublayers` collection should be populated automatically
- Note: requires the introduction of a `loaded` state (e.g. not-loaded / loading / loaded / error).
